docs/src/main/asciidoc/infinispan-client-guide.adoc

```diff
@@ -303,20 +364,12 @@ The Infinispan client supports both indexed and non indexed querying as long as
 properties of the proto schema.
 
 Query builds upon the proto definitions you can configure when setting up the `ProtoStreamMarshaller`.
-Make sure to configure this marshaller before attempting querying. Once this is completed
-you must add another optional depdency: `infinispan-query-dsl` to start using queries.
+Either method of Serialization above will automatically register the schema with the server at
+startup, meaning that you will automatically gain the ability to query objects store with
+the client.
 
 You can do this by adding the following dependency to your project.
 
-[source,xml]
-----
-<dependency>
-    <groupId>org.infinispan</groupId>
-    <artifactId>infinispan-query-dsl</artifactId>
-</dependency>
-----
-
-With this added you can use Infinispan querying just as you would normally, nothing special required.
```

Comment:
Either you don't remove that or you also remove the `You can do this by adding the following dependency to your project.` sentence but there's something wrong here.

Reply:
Oops missed removing the other.